src/Faker/Generator.php

```diff
@@ -139,13 +157,15 @@
  * @property string $languageCode
  * @property string $currencyCode
  * @property bool $boolean
+ *
  * @method bool boolean($chanceOfGettingTrue = 50)
  *
  * @property int    $randomDigit
  * @property int    $randomDigitNot
  * @property int    $randomDigitNotNull
  * @property string $randomLetter
  * @property string $randomAscii
+ *
```

Comment:
This is not ideal, but all of these annotations will be removed sooner or later.